client/components/navbar.jsx

```diff
@@ -1,17 +1,71 @@
 import React from 'react';
 import 'bootstrap-icons/font/bootstrap-icons.css';
```

Comment:
Same with the Bootstrap icons, move this import to the `App` component. When imported in the App component the icons will work in any component.